--- Makefile_orig	2020-12-06 19:29:08.348054177 +0000
+++ Makefile_mod	2020-12-06 19:30:03.647474329 +0000
@@ -19,10 +19,9 @@
 
 fancontrol:
 	go get -t -d -v ./main
-	go build $(BUILDINFO_STATIC) -p 4 main/fancontrol.go main/equations.go main/cputemp.go
+	go build $(BUILDINFO) -p 4 main/fancontrol.go main/equations.go main/cputemp.go
 
 xdump1090:
-	git submodule update --init
 	cd dump1090 && make
 
 xdump978:
